README.md

```diff
@@ -24,14 +24,17 @@ SABnzbd has a good deal of dependencies you'll need before you can get running.
 
 - `python` (only 2.7.x and higher, but not 3.x.x)
 - `python-cheetah`
-- `python-dbus`
```

Comment:
python-dbus, cryptography, unzip, 7zip and yenc are not required, SAB will work without them.
Just missing some features.
Now it's the same as in INSTALL.txt:
https://github.com/sabnzbd/sabnzbd/blob/master/INSTALL.txt